internal/cmd/rsl/record/record.go

```diff
@@ -28,7 +36,12 @@ func (o *options) Run(_ *cobra.Command, args []string) error {
 		return err
 	}
 
-	return repo.RecordRSLEntryForReference(args[0], true, rslopts.WithOverrideRefName(o.dstRef))
+	opts := []rslopts.Option{rslopts.WithOverrideRefName(o.dstRef)}
+	if o.skipDuplicateCheck {
```

Comment:
Let's track this in an issue because tag entry recording is a bit more complicated than just a duplicate check. What if a tag must be recreated? Do we allow that at all?